index.js

```diff
@@ -419,6 +419,55 @@
     return f(y, x);
   });
 
+  //# lift :: Apply f => (a -> b) -> f a -> f b
+  //.
+  //. Promotes a unary function to a function which operates on a Functor.
+  //.
+  //. ```javascript
+  //. > S.lift(R.inc, S.Just(2))
+  //. S.Just(3)
+  //.
+  //. > S.lift(R.inc, S.Nothing())
+  //. S.Nothing()
+  //. ```
+  S.lift = def('lift', [Function, a], R.map);
+
+  //# lift2 :: Apply f => (a -> b -> c) -> f a -> f b -> f c
+  //.
+  //. Promotes a binary function to a function which operates on two Apply's.
```

Comment:
Let's use `[Apply][]s`. I don't think an apostrophe is appropriate here (though I agree that "Applys" looks silly).